.github/workflows/release.yml

```diff
@@ -32,7 +32,7 @@ jobs:
     name: Build release tarball
     runs-on: ubuntu-latest
     container:
-      image: glotzerlab/ci:2022.08-ubuntu20.04
+      image: glotzerlab/ci:2022.11-ubuntu20.04
```

Comment:
Why was this change necessary? I see this in a few other places in this PR too.

Reply:
This pulls in the new Docker containers I built today with Python 3.11 installed: https://github.com/glotzerlab/ci-containers